src/mailbox_sender.cc

```diff
@@ -25,8 +26,10 @@ namespace base {
 MailboxSender::MailboxSender(const MailboxAddressBook& addr_book, zmq::context_t* zmq_context)
     : zmq_context_(zmq_context), addr_book_(addr_book) {
   addr_book_.ForEach([=](int process_id, const std::string& addr) {
+    std::cerr << "connect to " << addr << std::endl;
```

Comment:
ditto

Reply:
This one is for debug purpose. Sometime worker may fail in this scope.